README.md

```diff
@@ -1 +1,41 @@
-# ToDo-List
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome-Books-ES6
```

Comment:
- [ ] Kindly change the project name to `ToDo-List` instead of `Awesome-Books-ES6`.